apps/portal/src/content/docs/components/avatar.mdx

```diff
@@ -4,150 +4,170 @@ description: Avatar component
+      </div>
+    </div>
+    <div>
+      <h6 className="font-semibold mb-2 text-center">Initials (Default)</h6>
+      <div className="flex items-center justify-center gap-6">
+        <Avatar name="John Doe" size="sm" />
+        <Avatar name="John Doe" />
+        <Avatar name="John Doe" size="lg" />
+      </div>
+    </div>
+    <div>
+      <h6 className="font-semibold mb-2 text-center">Initials (Rounded)</h6>
+      <div className="flex items-center justify-center gap-6">
+        <Avatar name="John Doe" size="sm" rounded />
+        <Avatar name="John Doe" rounded />
+        <Avatar name="John Doe" size="lg" rounded />
+      </div>
+    </div>
+    <div>
+      <h6 className="font-semibold mb-2 text-center">Avatar Icon (Default)</h6>
+      <div className="flex items-center justify-center gap-6">
+        <Avatar size="sm" />
+        <Avatar />
+        <Avatar size="lg" />
+      </div>
+    </div>
+    <div>
+      <h6 className="font-semibold mb-2 text-center">Avatar Icon (Rounded)</h6>
+      <div className="flex items-center justify-center gap-6">
+        <Avatar size="sm" rounded />
+        <Avatar rounded />
+        <Avatar size="lg" rounded />
+      </div>
     </div>
 </div>`}
 />
 
 ## Usage
 
 ```typescript jsx
-import { Avatar } from '@harnessio/ui/components'
-
-//...
-
-return (
-  <Avatar.Root>
-    <Avatar.Image src="avatarurl.jpg" alt="Avatar image" />
-    <Avatar.Fallback>JD</Avatar.Fallback>
-  </Avatar.Root>
-)
+// Image (Default)
+<Avatar name="John Doe" src="/avatar.png" />
+<Avatar name="John Doe" src="/avatar.png" size="sm" />
+<Avatar name="John Doe" src="/avatar.png" size="lg" />
+
+// Image (Rounded)
+<Avatar name="John Doe" src="/avatar.png" rounded />
+<Avatar name="John Doe" src="/avatar.png" size="sm" rounded />
+<Avatar name="John Doe" src="/avatar.png" size="lg" rounded />
+
+// Initials (Default)
+<Avatar name="John Doe" />
+<Avatar name="John Doe" size="sm" />
+<Avatar name="John Doe" size="lg" />
+
+// Initials (Rounded)
+<Avatar name="John Doe" rounded />
+<Avatar name="John Doe" size="sm" rounded />
+<Avatar name="John Doe" size="lg" rounded />
+
+// Avatar Icon (Default)
+<Avatar />
+<Avatar size="sm" />
+<Avatar size="lg" />
+
+// Avatar Icon (Rounded)
+<Avatar rounded />
+<Avatar size="sm" rounded />
+<Avatar size="lg" rounded />
 ```
 
-## Anatomy
+## Sizes
 
-All parts of the `Avatar` component can be imported and composed as required.
+The `Avatar` component supports three sizes: `sm`, `default`, and `lg`.
 
-```typescript jsx
-<Avatar.Root>
-  <Avatar.Image />
-  <Avatar.Fallback />
-</Avatar.Root>
-```
-
-## API Reference
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="flex items-center gap-6">
+    <Avatar name="John Doe" src="/avatar.png" size="sm" />
+    <Avatar name="John Doe" src="/avatar.png" />
+    <Avatar name="John Doe" src="/avatar.png" size="lg" />
+</div>`}
+/>
 
-### `Root`
+## Variants
```

Comment:
These are not variants, maybe we could call this "Shapes" and update the description too?